### PR TITLE
fix frame drop percentage

### PR DESCRIPTION
**Pull request by haraschax**

### CODE REVIEW

File: selfdrive/modeld/modeld.cc [hunk 1]
```diff
@@ -193,17 +193,18 @@ int main(int argc, char **argv) {
             model_eval_frame(&model, q, yuv_ion.buf_cl, buf_info.width, buf_info.height,
                              model_transform, NULL, vec_desire);
         mt2 = millis_since_boot();
+        float model_execution_time = (mt2 - mt1) / 1000.0;
```
Comment (adeebshihadeh) [inline selfdrive/modeld/modeld.cc:196]: any reason this is in seconds?

Comment (HaraldSchafer) [reply selfdrive/modeld/modeld.cc:196]: Yes, want as much as possible in SI units.

File: selfdrive/modeld/models/driving.cc [hunk 2]
```diff
@@ -461,7 +466,9 @@ void model_publish(PubMaster &pm, uint32_t vipc_frame_id, uint32_t frame_id,
 }
 
 void posenet_publish(PubMaster &pm, uint32_t vipc_frame_id, uint32_t frame_id,
-                     uint32_t vipc_dropped_frames, float frame_drop, const ModelDataRaw &net_outputs, uint64_t timestamp_eof) {
+                     uint32_t vipc_dropped_frames, float frame_drop,
+                     const ModelDataRaw &net_outputs, uint64_t timestamp_eof,
+                     float model_execution_time) {
```
Comment (adeebshihadeh) [inline selfdrive/modeld/models/driving.cc:471]: this is unused

Comment (HaraldSchafer) [reply selfdrive/modeld/models/driving.cc:471]: fixed

**Pull request merged**